docker-compose.yml

```diff
@@ -16,8 +16,10 @@ services:
     env_file:
       - llm-server/.env
     ports:
-      - 8002:8002
-      - 5678:5678
+      - 8002
+      - 5678
```

Comment:
hey @codebanesr, now llm-server's flask app and debugpy server, both has dynamic public ports. Thanks for mentioning. 
![image](https://github.com/openchatai/OpenCopilot/assets/75121304/ac7248b7-dbf8-4839-a676-abdc04d7fcf5)